src/libraries/Common/src/Interop/Windows/HttpApi/Interop.HttpApi.cs

```diff
@@ -459,7 +459,7 @@ internal struct HTTP_BINDING_INFO
         internal static partial uint HttpInitialize(HTTPAPI_VERSION version, uint flags, IntPtr pReserved);
 
         [LibraryImport(Libraries.HttpApi, SetLastError = true)]
-        internal static partial uint HttpSetUrlGroupProperty(ulong urlGroupId, HTTP_SERVER_PROPERTY serverProperty, IntPtr pPropertyInfo, uint propertyInfoLength);
+        internal static unsafe partial uint HttpSetUrlGroupProperty(ulong urlGroupId, HTTP_SERVER_PROPERTY serverProperty, void* pPropertyInfo, uint propertyInfoLength);
```

Comment:
Unrelated change that I had around

Reply:
Just curious, does it save an extra validation? I always thought `SafeHandle` is expensive for P/Invokes and shouldn't be used unless native method requires it, but `IntPtr` in my mind was as good as `void*`. :)

Reply:
It's mostly for style reasons; now that `IntPtr` === `nint`, let's use an actual pointer type instead of a native integer when we pass a pointer. I don't think there are differences in execution behavior.